pilot/cmd/pilot-agent/status/server.go

```diff
@@ -432,7 +432,7 @@ func (s *Server) handleAppProbe(w http.ResponseWriter, req *http.Request) {
 		proberPath = "/" + proberPath
 	}
 	var url string
-	if prober.HTTPGet.Scheme == corev1.URISchemeHTTPS {
+	if prober.HTTPGet.Scheme == apimirror.URISchemeHTTPS {
```

Comment:
Now that we are removing the dependency of `corev1`, does it make sense to also update here?

https://github.com/istio/istio/blob/46df4bf95a5818afe54e54c70583b7429136a013/pkg/kube/inject/app_probe.go#L69-L71

Reply:
No, we only need to not import k8s library from the agent. Pilot need to import it anyways because it actually uses it

Reply:
specifically it would be even harder here since we would need to convert between the two for no real benefits